packages/react/src/types.ts

```diff
@@ -74,13 +74,12 @@ export type ClerkProp =
   | undefined
   | null;
 
-type ButtonProps = {
+type ButtonProps = React.PropsWithChildren<{
   afterSignInUrl?: string;
   afterSignUpUrl?: string;
   redirectUrl?: string;
   mode?: 'redirect' | 'modal';
-  children?: React.ReactNode;
```

Comment:
🤔  These are technically not the same, in `PropsWithChildren` `children` is required.

Reply:
<img width="804" alt="Screenshot 2023-11-15 at 23 54 47" src="https://github.com/clerk/javascript/assets/3936408/34ae51e5-c5b5-4c0f-8798-ecd471fa278c">

Reply:
OH! I guess I had it the wrong way around. The same comment stands for instances where children was not optional then. It probably doesn't matter in practice though 👍